model/textparse/openmetricslex.l.go

```diff
@@ -747,16 +735,58 @@ yyrule25: // {S}[^ \n]+
 		return tTimestamp
 	}
 yyrule26: // \n
-	{
+	if true { // avoid go vet determining the below panic will not be reached
 		l.state = sInit
 		return tLinebreak
 		goto yystate0
 	}
 	panic("unreachable")
 
-	goto yyabort // silence unused label error
-
 yyabort: // no lexem recognized
+	//
+	// silence unused label errors for build and satisfy go vet reachability analysis
```

Comment:
?? What created this changes in generated file?

Reply:
I executed the recommended command from [here](https://github.com/prometheus/prometheus/tree/main/model/textparse#making-changes-to-textparse-lexers) and that is the result I got. Don't you ?